src/components/TextField/TextField.tsx

```diff
@@ -120,10 +123,9 @@ export class TextField extends React.Component<ITextFieldProps, ITextFieldState>
                 {label && <Label htmlFor={this._id}>{label}</Label>}
                 {iconClass && <i className={iconClass}></i>}
                 {multiline ? this._renderTextArea() : this._renderInput()}
-                {(description || errorMessage) &&
+                {description &&
```

Comment:
can the description also be implemented as tooltip to save space around the component?